lib/kitchen/driver/dokken.rb

```diff
@@ -604,6 +607,9 @@ def wait_running_state(name, v)
       end
 
       def chef_container_name
+        if config[:platform] != ""
+          return "chef-#{chef_version}-" + config[:platform].sub("/", "-")
```

Comment:
@eheydrick I have found the issue! The reason why the same `chef/chef` container was used (with the wrong platform architecture) is because the container names are the same.

Reply:
Awesome! I'll give that a try.

Reply:
Works now!